### DISCUSSION TITLE
pobieranie etykiet dla kilku paczek jednocześnie przez API (WzA)

### DISCUSSION THREAD

**Discussion by ciaglok2** [allegro/allegro-api#5027] [Suggestions - resolved]
Jaki jest sens dostępu przez API do WzA bez tej funkcjonalności? Allegro mówi, że nie ma w planach jej wprowadzenia. Nie rozumiem, dlaczego. Jak sobie z tym radzicie?

**Comment by hahyes** [comment]
To jakiś szczególny problem, żeby wykonać zapytanie o każdą paczkę osobno?

**Reply by Onixarts** [reply]
Tak. Marnotrawienie papieru chociażby, brak możliwości wykorzystania etykiet A4x4 itp.

**Reply by hahyes** [reply]
> Tak. Marnotrawienie papieru chociażby, brak możliwości wykorzystania etykiet A4x4 itp.

@Onixarts Pobierz etykiety, połącz je i wydrukuj?

**Comment by ciaglok2** [comment]
Wysyłanie danych o paczce może się odbywać pojedynczo, bo to się da łatwo zutomatyzować, ale pobieranie etykiet pojedynczo? Musiałbym chyba zatrudnić dodatkową osobę do klikania i drukowania.

**Comment by vendoremo** [comment]
Popieram, brakuje takiej funkcji

**Comment by MartaNowaczyk** [comment]
Sugestię wdrożono wraz z nowym API dla WZA - [dokumentacja](https://developer.allegro.pl/documentation/#tag/Shipment-management).
